test/fx/quantization.py

```diff
@@ -240,7 +246,9 @@ def load_arg(a):
             if node.name in self.quants:
                 self.quants[node.name].observe(node, env)
 
-        return load_arg(self.graph.result)
+        if output_node is None:
```

Comment:
Technically this could just go into the `if op == 'output'` statement, and cause the entire loop to terminate and return. It skips the check that there is only one return node, but we should be enforcing that anyway.